community-operators/openshift-qiskit-operator/0.2.0/openshift-qiskit-operator.v0.2.0.clusterserviceversion.yaml

```
@@ -0,0 +1,331 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "qiskit.ibm.com/v1alpha1",
+          "kind": "QiskitPlayground",
+          "metadata": {
+            "name": "qiskitplayground-sample"
+          },
+          "spec": {
+            "image": "qiskit/jupyter:0.1",
+            "imagePullPolicy": "Always",
+            "pvc": "claim-admin",
+            "resources": {
+              "requests": {
+                "memory": "2G",
+                "cpu": "1"
+              },
+              "limits": {
+                "memory": "3G",
+                "cpu": "1.5"
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Developer Tools
+    certified: "false"
+    createdAt: 2021-004-23 22:25:53
+    containerImage: https://hub.docker.com/r/qiskit/operator
```

```suggestion
    containerImage: docker.io/qiskit/operator:0.1
```
I suppose this is your operator container image.

---

Done